packages/core/src/scenes/Random.ts

```diff
@@ -40,6 +40,28 @@ export class Random {
     return value;
   }
 
+  // This is an optimization.
+  // Since gauss() generates a pair of independent Gaussian random numbers,
+  // it returns one immediately and stores the other for the next call to gauss().
```

Comment:
I think these remarks should be written as TSDoc:
```suggestion
  /**
   * Previously generated Gaussian random number.
   *
   * @remarks
   * This is an optimization.
   * Since {@link gauss} generates a pair of independent Gaussian random
   * numbers, it returns one immediately and stores the other for the next call
   * to {@link gauss}.
   */
```